scripts/cli_smoke_tests.py

```diff
@@ -13,81 +13,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pathlib
 import subprocess
-import json
+import sys
 
 import click
 
-HERE = pathlib.Path(__file__).parent.resolve()
-SCRIPT_PATH = HERE.joinpath("test_data", "print_command_line.py")
-
 
 def main():
-    standard_cli = ["streamlit", "run", str(SCRIPT_PATH)]
-    _run_cli_smoke_tests(provided=standard_cli, expected=standard_cli)
+    standard_cli = ["streamlit", "version"]
```

Comment:
Smoke testing with `streamlit version` is probably not detecting the same set of problems compared to `streamlit run`?  Is the reason for using `streamlit version` that `streamlit run` did not work anymore without preheating?

Reply:
It actually ends up testing roughly the same thing (except the mechanism being used to test it is what I'm a bit unhappy about).

What previously happened is that the preheated script run would call a script that only prints the command that streamlit was called with, and the smoke test was to see if the program name `__main__.py` is correctly replaced with `streamlit`, even if streamlit was started by calling `python -m streamlit run ...` instead of `streamlit run ...` directly.

With the changes in this PR, `streamlit version` asserts that the program name is `streamlit`, so the smoke tests will fail if `streamlit version` and `python -m streamlit version` aren't equivalent.